doc/raketasks/backup_restore.md

```diff
@@ -360,8 +360,8 @@ If you are using backup restore procedures you might encounter the following war
 
 ```
 psql:/var/opt/gitlab/backups/db/database.sql:22: ERROR:  must be owner of extension plpgsql
-psql:/var/opt/gitlab/backups/db/database.sql:2931: WARNING:  no privileges could be revoked for "public" (two occurences)
-psql:/var/opt/gitlab/backups/db/database.sql:2933: WARNING:  no privileges were granted for "public" (two occurences)
```

Comment:
I found only one `occurences` inclusions in psql repository ([link to github search](https://github.com/postgres/postgres/search?utf8=✓&q=occurences&type=Code)). That's why I didnt' think that this typo from psql.